test/data/mini_example/mini_example.sam

```diff
@@ -0,0 +1,69 @@
+@SQ	SN:chr1	LN:368
+read001	0	chr1	1	60	50M	*	0	0	CGCCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCG	*	AS:i:50	NM:i:0
+read002	0	chr1	2	60	50M	*	0	0	GCCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGG	*	AS:i:50	NM:i:0
+read003	0	chr1	3	60	50M	*	0	0	CCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGC	*	AS:i:50	NM:i:0
+read004	0	chr1	4	60	50M	*	0	0	CCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCG	*	AS:i:50	NM:i:0
+read005	0	chr1	5	60	50M	*	0	0	CATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCGG	*	AS:i:50	NM:i:0
+read006	0	chr1	6	60	50M	*	0	0	ATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCGGC	*	AS:i:50	NM:i:0
+read007	0	chr1	7	60	50M	*	0	0	TGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCGGCA	*	AS:i:50	NM:i:0
+read008	0	chr1	8	60	50M	*	0	0	GCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCGGCAT	*	AS:i:50	NM:i:0
+read009	0	chr1	9	60	49M13D1M	*	0	0	CAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCGGCATA	*	AS:i:46	NM:i:13
+read010	0	chr1	10	60	48M13D2M	*	0	0	AACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCGGCATAT	*	AS:i:46	NM:i:13
+read011	0	chr1	11	60	47M13D3M	*	0	0	ACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCGGCATATA	*	AS:i:46	NM:i:13
+read012	0	chr1	12	60	46M13D4M	*	0	0	CTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCGGCGGCATATAC	*	AS:i:46	NM:i:13
+read013	0	chr1	51	60	7M13D43M	*	0	0	GCGGCATATACATAAGGGGCTCATCGATCGATTTCGGATCGGGGGGCCCC	*	AS:i:46	NM:i:13
+read014	0	chr1	52	60	6M13D44M	*	0	0	CGGCATATACATAAGGGGCTCATCGATCGATTTCGGATCGGGGGGCCCCC	*	AS:i:46	NM:i:13
+read015	0	chr1	53	60	5M13D45M	*	0	0	GGCATATACATAAGGGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCA	*	AS:i:46	NM:i:13
+read016	0	chr1	54	60	4M13D46M	*	0	0	GCATATACATAAGGGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCAT	*	AS:i:46	NM:i:13
+read017	0	chr1	55	60	3M13D47M	*	0	0	CATATACATAAGGGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCATT	*	AS:i:46	NM:i:13
+read018	0	chr1	56	60	2M13D48M	*	0	0	ATATACATAAGGGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCATTT	*	AS:i:46	NM:i:13
+read019	0	chr1	57	60	1M13D49M	*	0	0	TATACATAAGGGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCATTTT	*	AS:i:46	NM:i:13
+read020	0	chr1	71	60	50M	*	0	0	ATACATAAGGGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCATTTTA	*	AS:i:50	NM:i:0
+read021	0	chr1	80	60	46M4S	*	0	0	GGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCATTTTAAACGGCCCC	*	AS:i:46	NM:i:0	SA:Z:chr1,108,+,44S6M,60,6,0;
+read022	0	chr1	91	60	35M15S	*	0	0	TCGATTTCGGATCGGGGGGCCCCCATTTTAAACGGCCCCGGGGCCAATTT	*	AS:i:35	NM:i:0
+read027	0	chr1	94	60	4M46S	*	0	0	ATTTGGATCTTGACTCTGGAAAACTTTTAACGCCGGGAATCGGTAGTCCT	*	AS:i:4	NM:i:0
+read023	0	chr1	101	60	25M15I10M	*	0	0	ATCGGGGGGCCCCCATTTTAAACGGCCCCGGGGCCAATTTGGATCTTGACT	*	AS:i:31	NM:i:15
+read024	0	chr1	102	60	24M15I11M	*	0	0	TCGGGGGGCCCCCATTTTAAACGGCCCCGGGGCCAATTTGGATCTTGACTC	*	AS:i:31	NM:i:15
+read025	0	chr1	103	60	23M15I12M	*	0	0	CGGGGGGCCCCCATTTTAAACGGCCCCGGGGCCAATTTGGATCTTGACTCT	*	AS:i:31	NM:i:15
+read021	2048	chr1	108	60	44S6M	*	0	0	GGGCTCATCGATCGATTTCGGATCGGGGGGCCCCCATTTTAAACGGCCCC	*	AS:i:6	NM:i:0
```

Comment:
The two alignments of read021 overlap by 2bp which is not ideal.

Reply:
Yes, I saw that. I thougt I leave it here, as it is possible that this can happen. But I can replace the bases if you say its more confusing than helping.